src/api/status/public/style.css

```diff
@@ -1,5 +1,8 @@
-#results {
-  max-width: 600px;
+#sidenav-collapse-main > ul > li > a.active {
```

Comment:
This should be `.navlink.active`

Reply:
I tried this CSS with .navlink.active but it doesn't work. So, I edited.

Reply:
my bad, that was "nav-link ".

Reply:
I tried it but it makes like this.
![image](https://user-images.githubusercontent.com/50640730/142699916-376926de-b488-4cac-a3a2-36aebc9a38a0.png)